pkg/controllers/job/plugins/svc/svc.go

```diff
@@ -286,6 +300,23 @@ func (sp *servicePlugin) createNetworkPolicyIfNotExist(job *batch.Job) error {
 			},
 		}
 
+		if len(sp.networkPolicyIngressPortsSpecs) > 0 {
+			var ports []networkingv1.NetworkPolicyPort
+			for _, spec := range sp.networkPolicyIngressPortsSpecs {
+				portPolicy, err := parseNetworkPolicyPort(spec)
+				if err != nil {
+					klog.V(3).Infof("Failed to build network policy for Job <%s/%s>: %v", job.Namespace, job.Name, err)
```

Comment:
failed to parse network policy ports from plugin settings

Reply:
Done!